dll/cpl/mmsys/lang/pt-PT.rc

```diff
@@ -162,16 +170,16 @@ END
 
 IDD_SETUP1 DIALOGEX 0, 0, 227, 206
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
-CAPTION "Configuração"
+CAPTION "Definições"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "ReactOS Audio", 10243, 7, 5, 213, 192
+    GROUPBOX "ReactOS Áudio", 10243, 7, 5, 213, 192
     ICON 4393, 10241, 26, 26, 20, 20
-    LTEXT "Este computador não pode reproduzir áudio porque o serviço ReactOS Audio não está activado.", 10245, 60, 25, 150, 40
-    AUTOCHECKBOX "&Habilitar ReactOS Audio", 10253, 60, 68, 150, 9
+    LTEXT "Este computador não pode reproduzir áudio porque o serviço ReactOS Áudio não está activado.", 10245, 60, 25, 150, 40
+    AUTOCHECKBOX "&Habilitar ReactOS Áudio", 10253, 60, 68, 150, 9
     LTEXT "Notas", 10246, 58, 87, 150, 11
-    LTEXT "1. Só um administrador pode activar este serviço. Se não estiver ligado como administrador, será solicitado um nome e password de administrador.", 10247, 60, 100, 150, 40
-    LTEXT "2. Depois de activar o ReactOS Audio, terá de reiniciar o computador.", 10248, 60, 140, 150, 40
+    LTEXT "1. Só um administrador pode activar este serviço. Se não estiver ligado como administrador, será solicitado um nome e palavra-passe de administrador.", 10247, 60, 100, 150, 40
```

Comment:
Can you proved a screenshot of that dlg, to see whether that still fits the 150x40 with the longer string?

Reply:
You're probably right, but I can't find that dialog. Can you help me find this dialog box?

Reply:
Haha, good point. IDD_SETUP1 seems to be dead code indeed. It is not referenced from anywhere in any *.c file.
But you should still be able to preview it with the reshacker I guess.

Reply:
The IDD_SETUP* are for future functionality.